block-layer/codecs/dag-pb.md

```diff
@@ -0,0 +1,59 @@
+# DagPB Spec
+
+DagPB does not support the full ["IPLD Data Model."](../../data-model-layer/data-model.md)
+
+## Format
+
+The DagPB IPLD format is a legacy format implemented with a single protobuf.
```

Comment:
I don't think it's fair to call this legacy (yet) as it's what pretty much all IPFS data is encoded with and until `UnixFSv2` arrives there's no way to use anything else.